src/node/dns.ts

```diff
@@ -1,30 +1,70 @@
-import { default as dnsUtil } from 'node-internal:dns';
 import * as errorCodes from 'node-internal:internal_dns_constants';
+import {
+  getServers,
+  lookup,
+  lookupService,
+  resolve,
+  resolve4,
+  resolve6,
+  resolveAny,
+  resolveCname,
+  resolveCaa,
+  resolveMx,
+  resolveNaptr,
+  resolveNs,
+  resolvePtr,
+  resolveSoa,
+  resolveSrv,
+  setDefaultResultOrder,
+  getDefaultResultOrder,
+  setServers,
+} from 'node-internal:internal_dns';
+import * as dns from 'node-internal:internal_dns';
 
 export * from 'node-internal:internal_dns_constants';
+export {
+  getServers,
+  lookup,
+  lookupService,
+  resolve,
+  resolve4,
+  resolve6,
+  resolveAny,
+  resolveCname,
+  resolveCaa,
+  resolveMx,
+  resolveNaptr,
+  resolveNs,
+  resolvePtr,
+  resolveSoa,
+  resolveSrv,
+  setDefaultResultOrder,
+  getDefaultResultOrder,
+  setServers,
+} from 'node-internal:internal_dns';
+
+function toCallback<A = unknown, ReturnValue = unknown>(
+  fn: (arg: A) => Promise<ReturnValue>
+): (
+  input: A,
+  callback: (error: Error | null, value?: ReturnValue) => void
+) => void {
+  return (
+    input: A,
+    callback: (error: Error | null, value?: ReturnValue) => void
```

Comment:
Node.js will throw an error on these if `callback` is not a function, correct?

Reply:
Yes, definitely there is a `validateFunction` missing from here.